cluster/deploy.sh

```diff
@@ -62,18 +62,33 @@ $KUBECTL delete customresourcedefinitions vms.kubevirt.io  || :
 # Remove all external facing services
 externalServiceManifests | cluster/kubectl.sh --core delete -f - || :
 
-# Delete everything else
-for i in `ls manifests/*.yaml`; do
-    $KUBECTL delete -f $i --grace-period 0 2>/dev/null || :
-done
+if [[ -n "$DEPLOY_MANIFEST" ]]; then
```

Comment:
This if/else branch could be isolated to the create part.  The delete and external services sections are really shared between both.

basically
```
if [[ -n "$DEPLOY_MANIFEST" ]]; then
    $KUBECTL create -f $DEPLOY_MANIFEST
else
    for i in `ls manifests/*.yaml`; do
        $KUBECTL create -f $i
    done
fi
```

Reply:
Yes! I actually had it in mind, but …

Reply:
ha, but what? is there a reason this logic isn't good?